test/pages/completed_tasks_test.dart

```diff
@@ -50,5 +55,48 @@ void main() {
         expect(tileFinder, findsOneWidget);
       },
     );
+
+    testWidgets(
+      'Pop Up Occurs',
```

Comment:
"Pop Up Occurs" when what? I would name this something like "Pop-up occurs when the trash icon is tapped"

Reply:
done